processor/configuration/src/main/java/com/redhat/service/smartevents/processor/sources/SourceResolver.java

```diff
@@ -43,4 +43,9 @@ private String getBridgeWebhookUrl(String customerId, String bridgeId) {
             throw new GatewayProviderException("Can't find events webhook for bridge " + bridgeId);
         }
     }
+
+    @Override
+    public String getType() {
+        return "SourceResolver";
+    }
```

Comment:
```suggestion
    @Override
    public String getType() {
        return null;
    }

    @Override
    public boolean accept(String gatewayType) {
        return true;
    }
```

I think this captures better the fact that the `SourceResolver` has no specific type and accepts all the source types